database/migrations/default/1730482587632_fix_is_manual_geocode/up.sql

```diff
@@ -0,0 +1,101 @@
+-- public.crashes_list_view source
+
+-- recreating this view to correct the is manual geocode column
+CREATE OR REPLACE VIEW public.crashes_list_view
+AS WITH geocode_status AS (
+         SELECT cris.id,
+            unified.latitude IS NOT NULL AND unified.latitude IS DISTINCT FROM cris.latitude OR unified.longitude IS NOT NULL AND unified.longitude IS DISTINCT FROM cris.longitude AS is_manual_geocode
```

Comment:
`<>` cannot treat `null` as a comparable value, `IS DISTINCT FROM` can treat `null` as as a comparable value, this was the root of the bug :)